addons/binding/org.openhab.binding.onebusaway/ESH-INF/thing/route.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onebusaway"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="route">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="stop" />
+        </supported-bridge-type-refs>
+        <label>OneBusAway Route</label>
```

Comment:
Then just call it "Route". As it is a route of a bus/subway/railway, but not that of a OneBusAway.